pom.xml

```diff
@@ -241,5 +242,23 @@
             <version>5.2.9.RELEASE</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+            <version>1.3.2</version>
+            <scope>provided</scope>
+        </dependency>
     </dependencies>
+    
+    <profiles>
+        <profile>
+            <id>active-on-jdk-11</id>
+            <activation>
+                <jdk>[11,</jdk>
+            </activation>
+            <properties>
+                <sling.java.version>11</sling.java.version>
```

Comment:
we cross-compile, that means, we always target Java 8 (even on JDK11). Please remove this profile!